workshops/templates/workshops/event.html

```diff
@@ -20,7 +20,17 @@
   <tr><td>Website URL:</td><td colspan="2">{{ event.website_url|default:"—"|urlize_newtab }} {% if event.url %}<a href="{% url 'validate_event' event.get_ident %}" class="btn btn-primary btn-xs pull-right" id="validate_event">validate event</a>{% else %}<a class="btn btn-danger btn-xs pull-right" id="error_event_url" href="#" data-toggle="popover" data-trigger="focus" title="Validation error" data-content="Cannot validate an event without URL pointing to the GitHub repository, e.g.: <code>https://github.com/swcarpentry/2015-05-24-training</code>">Error</a>{% endif %}</td></tr>
   <tr><td>Eventbrite key:</td><td colspan="2">{% if event.reg_key %}<a href="https://www.eventbrite.com/myevent?eid={{ event.reg_key }}" title="Go to Eventbrite's page for this event" target="_blank">{{ event.reg_key }}</a>{% else %}—{% endif %}</td></tr>
   <tr><td>admin fee:</td><td colspan="2">{{ event.admin_fee|default_if_none:"—" }}</td></tr>
-  <tr><td>invoice:</td><td colspan="2">{{ event.get_invoice_status_display }}</td></tr>
+  <tr>
+    <td>invoice:</td>
+    <td colspan="2">
+      {{ event.get_invoice_status_display }}
+      {% if event.invoice_status == 'not-invoiced' %}
```

Comment:
Hm - decorate event in the view class and then use `{% if event._show_invoice_link %}` or something like that?  Feels like this logic could change frequently.